lib/Mage/Backup/Filesystem.php

```diff
@@ -87,13 +87,14 @@ public function rollback()
         $rollbackWorker->run();
 
         $this->_lastOperationSucceed = true;
+        return $this->_lastOperationSucceed;
     }
 
     /**
      * Implementation Create Backup functionality for Filesystem
      *
      * @throws Mage_Exception
-     * @return boolean
+     * @return bool
```

Comment:
It can only return true?

```suggestion
     * @return true
```